dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/AbstractConfig.java

```diff
@@ -57,7 +57,7 @@
 
     private static final Pattern PATTERN_PATH = Pattern.compile("[/\\-$._0-9a-zA-Z]+");
 
-    private static final Pattern PATTERN_NAME_HAS_SYMBOL = Pattern.compile("[:*,/\\-._0-9a-zA-Z]+");
+    private static final Pattern PATTERN_NAME_HAS_SYMBOL = Pattern.compile("[:*,\\s/\\-._0-9a-zA-Z]+");
```

Comment:
I just learned that this is a limitation from the Dubbo framework. Previously, I got an exception when trying to use mock value with the following format:
`fail: return null`